File: myblog-server/src/main/java/com/site/blog/model/entity/SysDictType.java
package com.site.blog.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;

/**
 * @author yanni
 * @date time 2022/6/17 12:33
 * @modified By:
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SysDictType {

    @TableId(type = IdType.ASSIGN_ID)
    String id;
    String name;
    String code;
    Integer sort;
    String remark;
    Boolean status;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    LocalDateTime createTime;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    LocalDateTime updateTime;
}

File: BlogApi/BlogApi/Models/SysDictType.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BlogApi.Models;

[Table("sys_dict_type")]
public class SysDictType
{
    [Key]
    [Column("id")]
    public string Id { get; set; } = Guid.NewGuid().ToString("N");

    [Column("name")]
    public string Name { get; set; } = string.Empty;

    [Column("code")]
    public string Code { get; set; } = string.Empty;

    [Column("sort")]
    public int Sort { get; set; }

    [Column("remark")]
    public string? Remark { get; set; }

    [Column("status")]
    public bool Status { get; set; }

    [Column("create_time")]
    public DateTime? CreateTime { get; set; }

    [Column("update_time")]
    public DateTime? UpdateTime { get; set; }
}
